app/SmartAlbums/BaseSmartAlbum.php

```diff
@@ -73,6 +74,8 @@ protected function __construct(SmartAlbumType $id, \Closure $smart_condition)
 			throw new FrameworkException('Laravel\'s service container', $e);
 		}
 		// @codeCoverageIgnoreEnd
+
+		parent::__construct();
```

Comment:
I am not sure about that one.

Reply:
I note this one, was trying to make phpstan happy, I will come back to this

Reply:
https://github.com/LycheeOrg/Lychee/blob/master/app/Models/BaseAlbumImpl.php#L33 here is some reading :)